ts/parsers/slideParser.ts

```diff
@@ -1,14 +1,141 @@
 import * as format from "string-template";
 import { getAttributeByPath, ZipHandler } from "../helpers";
-import { GraphicFrameParser, PowerpointElementParser } from "./";
+import { GraphicFrameParser, PowerpointElementParser, SlideRelationsParser } from "./";
 
 export default class SlideParser {
-    public static async getSlideElements(PPTElementParser: PowerpointElementParser, slideNumber): Promise<any[]> {
+    public static async getSlideLayout(slideRelations) {
+        // Read relationship filename of the slide (Get slideLayoutXX.xml)
+        // @sldFileName: ppt/slides/slide1.xml
+        // @resName: ppt/slides/_rels/slide1.xml.rels
+        let relationshipArray = slideRelations["Relationships"]["Relationship"];
+        let layoutFilename = "";
+        if (relationshipArray.constructor === Array) {
```

Comment:
`Array.isArray(relationshipArray)` doesn't work? There's also lodash array check if you have access to it already

Reply:
changed it to isArray, i don't have the lodash one installed. so using isArray now